sdk/sdk-api/src/main/java/mil/emp3/api/Camera.java

```diff
@@ -1,12 +1,13 @@
 package mil.emp3.api;
 
-import org.cmapi.primitives.GeoBase;
 import org.cmapi.primitives.GeoCamera;
 import org.cmapi.primitives.IGeoAltitudeMode;
 import org.cmapi.primitives.IGeoCamera;
+import org.cmapi.primitives.IGeoPosition;
 
 import java.security.InvalidParameterException;
```

Comment:
Would it better to use IllegalArgumentException instead of IllegalStateException? We shouldn't be using InvalidParameterException as that is under securiy package?